CONTRIBUTING.md

```diff
@@ -0,0 +1,58 @@
+# Contributing
+
+:+1::tada: First off, thanks for taking the time to contribute! :tada::+1:
+
+We welcome contributions via both the raising of issues and submitting pull requests. But before you
+do, please take a moment to consult the below.
+
+## Chatting
+
+If before any of the below you'd like to discuss an issue or some code, then come and chat with
+us on one of the following platforms:
+
+* [Users mailing list](https://groups.google.com/a/apereo.org/forum/#!forum/equella-users) - Most
+  active, best for more usage centric questions
+* [Developer mailing list](https://groups.google.com/a/apereo.org/forum/#!forum/equella-dev) - For
+  discussing work on the code
+* \#Equella channel in [Apereo's slack](https://apereo.slack.com/)
+
+## Submitting Issues
+
+Issues relating to the documentation are welcome. (For any issues relating to openEQUELLA itself,
+please switch over to its repo.)
+
+When raising issues, please try and be as detailed as possible. Links, line numbers, etc always
```

Comment:
It's nice to document this, but most new contributors will ignore this.
Issue templates is an easier way to nudge folks in the right direction.
https://help.github.com/articles/about-issue-and-pull-request-templates
https://help.github.com/articles/creating-issue-templates-for-your-repository

E.G. https://github.com/Jasig/uPortal/tree/master/.github/ISSUE_TEMPLATE

Reply:
Something similar to https://github.com/equella/Equella/pull/607 maybe?

Reply:
Being that this is a minor repo (and not even code - so there's not much we can enforce), happy with the soft suggestion. Prefer contribution over process in this one.

Reply:
I wouldn't say no to a simple issue template PR, but this is sufficient as well.